### ISSUE TITLE
Need to decide if a turn is present

### ISSUE THREAD

**Issue by IanMayo** [billwiliams/Non_linear_regression#4]
Currently, we presume that there is a turn, and we just find the slice with the minimum score.

But, there may not have been a turn.  I think there is only a turn when one set of error sums are much, much smaller than the others.  Where there isn't a clear winner, there probably isn't a turn.

We need to come up with an algorithmic way of doing this.

**Comment by billwiliams** [comment]
true. a turn would be presumed when one set of error sums are considerably smaller than the others
